components/molecule/textareaField/src/index.js

```diff
@@ -29,6 +29,7 @@ const MoleculeTextareaField = ({
   autoHideHelpText = false,
   helpText,
   value = '',
+  updateInternalValue = '',
```

Comment:
I don't get why we need to props, `value` and `updateInternalValue`. If what we want is a controlled component we should expose only two props: `value` and `onChange` and if both of these props are not properly defined, I think that we can assume that the client want to use the uncontrolled version.

Reply:
I agree with you @carlosvillu but I debated this with @jcger and we decided to create a new prop in order to ensure the retro-compatibility of the compontent.

Reply:
And what about creating a new (major) version with only `value` and `onChange` properly used instead of adding layers of props? @carlosvillu  @andresin87

Reply:
@arnau-rius the final version will only contain `value `and `onChange`, so not `updateInternalValue` nor `inicialValue`, I'm working on it right now

Reply:
good news!